scripts/manage.py

```diff
@@ -33,12 +33,25 @@ def unclaimed(network, start_block, address):
 @account_option()
 @click.option("--start-block", type=int)
 @click.option("--batch-size", type=int, default=256)
+@click.option("--multicall/--no-multicall", "use_multicall", default=True)
 @click.argument("address", type=AddressType)
-def claim(network, account, start_block, batch_size, address):
+def claim(account, start_block, batch_size, use_multicall, address):
     """Claim unclaimed streams using multicall (anyone can claim)"""
 
     sm = StreamManager(address=address)
     unclaimed_streams = sm.unclaimed_streams(start_block=start_block)
+
+    if not use_multicall:
+        for _ in range(batch_size):
+            try:
+                stream = next(unclaimed_streams)
+            except StopIteration:
+                return
+
+            stream.claim(sender=account)
+
+        raise click.UsageError("More claims needed")
```

Comment:
suggestion(non-blocking): Maybe change this to something like "There are more unprocessed claims"? Or maybe even not raise an exception at all, just exit clean.

Reply:
Yeah, I think no need to raise, but should just give a a nice reminder that there are more claims to do

Reply:
22c9527